src/domain/alerts/mappers/multi-send.mapper.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@nestjs/common';
+import {
+  decodeFunctionData,
+  getAddress,
+  Hex,
+  hexToBigInt,
+  hexToNumber,
+  parseAbi,
+  size,
+  slice,
+} from 'viem';
+
+@Injectable()
+export class MultiSendMapper {
+  private readonly abi = parseAbi([
+    'function multiSend(bytes memory transactions)',
+  ]);
+
+  mapMultiSendTransactions(multiSendData: Hex): Array<{
+    operation: number;
+    to: string;
+    value: bigint;
+    data: Hex;
+  }> {
+    const mapped: Array<{
+      operation: number;
+      to: string;
+      value: bigint;
+      data: Hex;
+    }> = [];
+
+    const multiSend = decodeFunctionData({
+      abi: this.abi,
+      data: multiSendData,
+    });
+    const transactions = multiSend.args[0];
+
+    let i = 0;
+
+    while (i < size(transactions)) {
+      // uint8 operation, address to, value uint256, dataLength uint256, bytes data
+      const operation = slice(transactions, i, (i += 1));
+      const to = slice(transactions, i, (i += 20));
+      const value = slice(transactions, i, (i += 32));
+      const dataLength = slice(transactions, i, (i += 32));
```

Comment:
I'd extract these numeric values to class variables instead, something like:
```
private static readonly OPERATION_END = 1;
private static readonly TO_END = 20;
private static readonly VALUE_END = 32;
private static readonly DATA_LENGTH_END = 32;
```

Wdyt?

Reply:
It's much more readable. I've moved them in f78093192d8c7d5555f13003c7548c368e3a20b0, opting for `*_SIZE` to be consistent.